OptimoveSDK/Sources/Classes/Optimobile/OptimoveInApp.swift

```diff
@@ -3,7 +3,7 @@
 import Foundation
 import CoreData
 
-public class InAppInboxItem {
+@objc public class InAppInboxItem :NSObject {
```

Comment:
This looks good but are we missing any `@objc` annotations / `NSObject` subclassing on the public APIs in here (`OptimoveInApp`) (and the ones added to the Optimove class itself?)

Reply:
added annotations to OptimoveInApp
Optimove class looks fine annotations wise